MdeModulePkg/Universal/Variable/RuntimeDxe/RuntimeDxeUnitTest/BlackBoxTest/Guid.c

```diff
@@ -0,0 +1,204 @@
+/** @file
+
+  Copyright 2006 - 2016 Unified EFI, Inc.<BR>
+  Copyright (c) 2010 - 2018, Intel Corporation. All rights reserved.<BR>
+
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
```

Comment:
I assume this was copied from an old test suite or something. It should be updated to use the new SPDX copyright format though. The block comment below is also older style and could be folded into a single comment.

Reply:
Similar for other files.

Reply:
Updated all new files to the new SPDX identifier format